mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_folder_mute_config_test.go

```diff
@@ -0,0 +1,127 @@
+					resource.TestCheckResourceAttr(
+						"google_scc_v2_folder_mute_config.default", "folder_mute_config_id", fmt.Sprintf("tf-test-my-config%s", contextBasic["random_suffix"])),
+					resource.TestCheckResourceAttr(
+						"google_scc_v2_folder_mute_config.default", "location", contextBasic["location"].(string)),
+					resource.TestCheckResourceAttr(
+						"google_scc_v2_folder_mute_config.default", "parent", fmt.Sprintf("folders/%s", contextBasic["folder_id"])),
+				),
+			},
+			{
+				ResourceName:            "google_scc_v2_folder_mute_config.default",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"parent", "folder_mute_config_id"},
+			},
+			{
+				Config: testAccSecurityCenterV2FolderMuteConfig_highSeverity(contextHighSeverity),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						"google_scc_v2_folder_mute_config.default", "description", "A test folder mute config with high severity"),
+					resource.TestCheckResourceAttr(
+						"google_scc_v2_folder_mute_config.default", "filter", "severity = \"HIGH\""),
+					resource.TestCheckResourceAttr(
+						"google_scc_v2_folder_mute_config.default", "folder_mute_config_id", fmt.Sprintf("tf-test-my-config%s", contextHighSeverity["random_suffix"])),
+					resource.TestCheckResourceAttr(
+						"google_scc_v2_folder_mute_config.default", "location", contextHighSeverity["location"].(string)),
+					resource.TestCheckResourceAttr(
+						"google_scc_v2_folder_mute_config.default", "parent", fmt.Sprintf("folders/%s", contextHighSeverity["folder_id"])),
+				),
+			},
+			{
+				ResourceName:            "google_scc_v2_folder_mute_config.default",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"parent", "folder_mute_config_id"},
+			},
+		},
+	})
+}
+
+func testAccSecurityCenterV2FolderMuteConfig_basic(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_scc_v2_folder_mute_config" "default" {
+  description          = "A test folder mute config"
+  filter               = "severity = \"LOW\""
+  folder_mute_config_id = "tf-test-my-config%{random_suffix}"
+  location             = "%{location}"
+  parent               = "folders/%{folder_id}"
+}
+`, context)
+}
+
+func testAccSecurityCenterV2FolderMuteConfig_highSeverity(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_scc_v2_folder_mute_config" "default" {
+  description          = "A test folder mute config with high severity"
+  filter               = "severity = \"HIGH\""
+  folder_mute_config_id = "tf-test-my-config%{random_suffix}"
+  location             = "%{location}"
+  parent               = "folders/%{folder_id}"
+}
+`, context)
+}
+func testAccCheckSecurityCenterV2FolderMuteConfigDestroyProducer(t *testing.T) resource.TestCheckFunc {
```

Comment:
This function is included in the generated test and doesn't need to be in the handwritten test.

https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-10772-old..auto-pr-10772#diff-f633485673f086d9d9aa703ccee741f8d664d7db68cdd8634f20ce90cc65c461R75

Reply:
addressed